cmd/environment.go

```diff
@@ -46,16 +47,17 @@ DESCRIPTION
 }
 
 type Environment struct {
-	Version     string
-	GitRevision string
-	SpecVersion string
-	SocatImage  string
-	TarImage    string
-	Languages   []string
-	Templates   map[string][]string
-	Environment []string
-	Cluster     string
-	Defaults    config.Global
+	Version         string
+	GitRevision     string
+	SpecVersion     string
+	SocatImage      string
+	TarImage        string
+	Languages       []string
+	DefaultBuilders map[string]string
```

Comment:
As @matejvasek pointed out, you should also include the buildpack default builders.
```suggestion
	DefaultBuilders map[string]map[string]string
```

Reply:
PTAL i have added buildpack

Reply:
Looks great! I just have one minor request for change.